dashdblocal_notebooks/livy-notebooks/IoT_Demo_Pyspark.ipynb

```diff
@@ -0,0 +1,1246 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# IoT Streaming Analytics Demo\n",
+    "This is a demo notebook for dashDB's integrated Apache Spark environment and how it can be used to process and land streaming data in dashDB tables.\n",
+    "\n",
+    "If you are new to dashDB and it's integrated Apache Spark capabilities you may first want to check out this [overview article](http://www.ibmbigdatahub.com/blog/evolving-enterprise-data-warehouse-beyond-sql-apache-spark)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![](https://ibm.box.com/shared/static/6dr6gou6xr1alfwnj6zkj0tbsyc64vda.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set up IoT demo data producer\n",
+    "This demo relies on a little data producer container that you have to set up in addition to your Db2 local and Jupyter notebook containers on the same host machine. You can find it [here](https://github.com/ibmdbanalytics/dashdb_analytic_tools/tree/notebook-dev/dashdblocal_notebooks/iot_producer) in a sub folder of the dashDB Jupyter notebook container project. Please follow the instructions found there and perform the few simple steps to set it up. \n",
+    "\n",
+    "This producer will set up a Kafka server on port `9092` with a Kafka topic named `iot4dashdb` and permanently flow in there messages about wind turbine device measurements."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__How to start the iotproducer container__\n",
```

Comment:
There's no need to cover these instructions in this notebook. https://github.com/ibmdbanalytics/dashdb_analytic_tools/tree/notebook-dev/dashdblocal_notebooks/iot_producer already has all the information and the non-Livy IoT demo (https://github.com/ibmdbanalytics/dashdb_analytic_tools/blob/master/dashdblocal_notebooks/IoT_demo.ipynb) only refers to that.

Reply:
isn't it more practical to have everything in one place? plus the second link is a scala notebook whereas this is a Python notebook

Reply:
Let me answer your questions in the opposite order: The second link is a Scala notebook, yes, but its content is basically the same as your Python notebook - it uses the same libraries and the same input, ... . Ideally, as a customer, I would be able to just read them side-by-side and have them look the same, except for the syntactical differences between the two languages. So they should also contain the same (or nearly the same) setup instructions, including how to set up the iot_producer container.

Reply:
ok I've deleted the corresponding cell